content/docs/command-reference/exp/index.md

```diff
@@ -0,0 +1,44 @@
+# exp[eriments]
+
+A set of commands to generate and compare local _experiments_:
```

Comment:
How about
```suggestion
A set of commands to generate and manage <abbr>experiments</abbr>:
```
? (which would also need a glossary entry in content\docs\user-guide\basic-concepts)

Reply:
p.s. The keyword/term here being "experiment management".

Reply:
BUT this brings up another issue which is: how do we clearly distinguish between regular experiments (e.g. in https://dvc.org/doc/start/experiments) and the special experiments enabled by this feature? Do we need to come up with terminology like "DVC experiment" and use it consistently (along with the tooltip I suggested above)? Cc @shcheklein

Reply:
The terminology is definitely an issue, we could maybe use the "ephemeral experiment vs formal/committed experiment" wording that was suggested way back in the original discussion.

Reply:
Briefly discussed this with Ivan and it turns out there may not be a need to disambiguate: all experiments are experiments, it's just that some are committed. It's more a distinction in the way they are managed, but we can leave that explanation to an "Experiment Management" guide or use case that we should write (not in this PR of course).